electron/utils.ts

```diff
@@ -41,6 +41,11 @@ async function isEpicServiceOffline(
   type: 'Epic Games Store' | 'Fortnite' | 'Rocket League' = 'Epic Games Store'
 ) {
   const epicStatusApi = 'https://status.epicgames.com/api/v2/components.json'
+  const notification = new Notification({
```

Comment:
Perhaps add a timeout for it with a bigger duration than the default.
Also, check if the i18n keys will be generated when using it inside a string literal like you are doing. If not, you need to add the keys as a comment in the code.
run `yarn i18n` and check if they keys were created.

Reply:
String are generated. Can't set timeout time.